jarviscli/packages/timeIn.py

```diff
@@ -1,16 +1,28 @@
 # -*- coding: utf-8 -*-
+import os
+import shutil
 import json
 import requests
 from colorama import Fore
 
+# this sets the path to the modules directory not the directory it was call from
+module_path = os.path.dirname(__file__)
+
 
 def main(self, s):
     # Trims input s to be just the city/region name
     s = s.replace('time ', '').replace('in ', '')
+    exists = os.path.isfile(module_path + '/../data/keys.json')
+    if not exists:
+        shutil.copy2(module_path + '/../data/samplekeys.json', module_path + '/../data/keys.json')
```

Comment:
Don't think we should do that here - more API keys might come and that would duplicate this code. Possible places would probably be:
* setup.sh
* ``__main__.py``

Reply:
agree with this one, I thought of it but was unable to implement it in a nice way :(